apps/website/core-components/select/README.md

```diff
@@ -77,3 +77,87 @@ A common strategy is to combine an input field with a select box so that a user
     </cds-form-group>
     </div>
 </DocIndent>
+
+::: component-section-level-one-title
+
+## Layouts
+
+:::
+
+::: component-section-level-one
+
+Select is available in vertical, horizontal, compact, and multiple layout. The select multiple option enables to user to select and view several items at once. See more layout guidelines on the [form page](/web-components/form/).
```

Comment:
`Select elements are` not `Select is`

`Select elements with the multiple attribute` not `The select multiple option`

`enables the user` not `enables to user`